src/components/GrowerDetail.js

```diff
@@ -93,7 +93,10 @@ const GrowerDetail = (props) => {
               setGrowerRegistrations(sortedRegistrations);
               setDeviceIdentifiers(
                 sortedRegistrations
-                  .map((reg) => reg.device_identifier)
+                  .map((reg) => ({
+                    id: reg.device_identifier,
+                    os: reg.manufacturer === 'apple' ? 'iOS' : 'Android',
```

Comment:
In the test DB, the manufacturer has an uppercase A for Apple. To be safe, it might be worth casting to lowercase before comparing.